package.json

```diff
@@ -17,7 +17,7 @@
     "build:docs": "yarn workspaces foreach --parallel --interlaced --verbose run build:docs",
     "build:watch": "yarn run build --watch",
     "changelog:validate": "yarn workspaces foreach --parallel --interlaced --verbose run changelog:validate",
-    "child-workspace-package-names-as-json": "ts-node scripts/child-workspace-package-names-as-json.ts",
+    "child-workspace-packages-as-json": "yarn workspaces list --json | jq --slurp 'map(select(.location != \".\"))'",
```

Comment:
I've removed this script as `jq` already does what we want (but better).

However, this does now require that people have `jq` installed. I think it's a useful tool, but it's not a requirement for this project at the moment. We could make it one, though. What do you think?